core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/autoscales/AutoscaleV4Endpoint.java

```diff
@@ -162,6 +163,7 @@ void stopInstancesForClusterCrn(@PathParam("crn") String clusterCrn,
             notes = Notes.STACK_NOTES, nickname = "autoscaleStopInstancesByName")
     void stopInstancesForClusterName(@PathParam("name") String clusterName,
             @RequestBody @NotEmpty List<String> instanceIds,
+            @QueryParam("hostGroup") String hostGroup,
```

Comment:
Same... does the hostGroup need to be specified here?

Reply:
Same justification as above comment. Added this param to these endpoints so that the CLI can propagate this info to CB